### PR TITLE
[DA-140]: Compare JSON objects by value in tests

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

**Comment by thescouser89** [comment]
:+1:

**Comment by jbartece** [comment]
Rebased to master

**Comment by jbartece** [comment]
retest this please

File: testsuite/src/test/java/org/jboss/da/test/client/AbstractRestApiTest.java [hunk 1]
```diff
@@ -202,4 +204,19 @@ private String convertContentType(ContentType contentType) {
...
+        } catch (JSONException ex) {
+            fail("The test wasn't able to compare JSON strings" + ex);
+        }
+    }
+
+    @Test
+    public void testJsonEquals() throws JSONException {
+        String s1 = "[{\"groupId\": \"com.google.guava\", \"artifactId\": \"guava\", \"version\": \"13.0.1\"}]";
+        String s2 = "[{\"groupId\": \"com.google.guava\", \"version\": \"13.0.1\", \"artifactId\": \"guava\"}]";
+        assertEqualsJson(s1, s2);
```
Comment (janinko) [inline testsuite/src/test/java/org/jboss/da/test/client/AbstractRestApiTest.java]: :100:

**Comment by janinko** [comment]
:+1:

**Pull request merged**